src/db/table/transactions.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `transactions` (
+  `transaction_id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `transaction_type_id` int(11) DEFAULT NULL,
+  `transaction_amount` decimal(15,2) NOT NULL,
+  `user_id` int(11) DEFAULT NULL,
```

Comment:
* Feel free to prepend context on foreign key columns. Eg. "sender_user_id" if applicable. 
* Nit: Please group foreign key columns together (adjacent columns)